Source/Core/VideoCommon/Assets/ShaderAsset.cpp

```diff
@@ -45,28 +49,38 @@ bool ParseShaderProperties(const VideoCommon::CustomAssetLibrary::AssetID& asset
     std::transform(type.begin(), type.end(), type.begin(),
                    [](unsigned char c) { return std::tolower(c); });
```

Comment:
Actually now that I see this, probably open a separate PR for this but, `std::tolower` is locale-dependent, probably don't want that. We have a locale-independent variant somewhere in Common IIRC.

Reply:
Yep, I was noticing that a bit ago.  I'll open a PR for that, thanks

Reply:
`TextureAsset.cpp` has the same issue.

Reply:
Looks like I caught it in Material.  Maybe you mentioned it there and I didn't consider the others.  I'll go hunting.  Thanks!